src/components/Anchor/Anchor.js

```diff
@@ -163,7 +169,9 @@ class Anchor extends React.Component {
   };
 
   handleClick = event => {
-    const { onClick, to } = this.props;
+    const { onClick, to, disabled } = this.props;
+    // Don't handle clicks if disabled
+    if (disabled) return;
```

Comment:
Does this need an `event.preventDefault`?  It seems to work from styleguide, but I'm not certain

Reply:
I'm not sure we need to do this at all if we apply `pointer-events: none` on the anchor when it's disabled.